presto-main/src/main/resources/webapp/assets/plan.js

```diff
@@ -299,6 +312,12 @@ let LivePlan = React.createClass({
                         { this.renderProgressBar() }
                     </div>
                 </div>
+                <hr className="h3-hr"/>
```

Comment:
By the way, this line is a regression - adds an extra line to the UI for no reason, and makes it inconsistent with the other pages.